core-services/secret-mirroring/_mapping.yaml

```diff
@@ -264,3 +264,9 @@ secrets:
   to:
     namespace: ci
     name: deployment-validation-operator-codecov-token
+- from:
+    namespace: psap
+    name: psap-entitlement-secret
+  to:
+    namespace: ci
```

Comment:
if you are planning to use this secret in your multi-stage jobs you should probably want to send it to `test-credentials` namespace, because of the RBACs that already exist.

Reply:
I am planning on using them here https://github.com/openshift/release/blob/master/ci-operator/config/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master.yaml#L25

Reply:
64f1099